Doc/whatsnew/2.3.rst

```diff
@@ -218,10 +218,10 @@ queen threatens another) and the Knight's Tour (a route that takes a knight to
 every square of an $NxN$ chessboard without visiting any square twice).
 
 The idea of generators comes from other programming languages, especially Icon
-(https://www.cs.arizona.edu/icon/), where the idea of generators is central.  In
+(https://www2.cs.arizona.edu/icon/), where the idea of generators is central.  In
```

Comment:
Please keep the original link. While it's currently a redirection to https://www2.cs.arizona.edu/icon/ we can still find https://www.cs.arizona.edu/icon/ on the site itself, but also on Wikipedia.